controllers/admin/AdminAdyenOfficialPrestashopCronController.php

```diff
@@ -93,15 +93,25 @@ public function __construct()
         $this->context = \Context::getContext();
 
         parent::__construct();
-        $this->postProcess();
-        die();
+        $failedNotifications = $this->postProcess();
+        if (empty($failedNotifications)) {
+            $message = 'Cron job finished successfully';
+        } else {
+            $message = sprintf(
+                'An error occurred during the execution of the following notifications: %s',
+                implode(',', $failedNotifications)
```

Comment:
This would read better I believe
```suggestion
                implode(', ', $failedNotifications)
```